packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1642,10 +1654,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             message: extractSummaryMetadataMessage(this.deltaManager.lastMessage) ?? this.messageAtLastSummary,
             sessionExpiryTimeoutMs: this.garbageCollector.sessionExpiryTimeoutMs,
         };
+        addBlobToSummary(summaryTree, metadataBlobName, JSON.stringify(metadata));
     }
 
-    private addContainerStateToSummary(summaryTree: ISummaryTreeWithStats, telemetryContext?: ITelemetryContext) {
-        addBlobToSummary(summaryTree, metadataBlobName, JSON.stringify(this.formMetadata()));
+    private addContainerStateToSummary(summaryTree: ISummaryTreeWithStats) {
```

Comment:
@sonalivdeshpande This method is supposed to have optional param `telemetryContext` to also be used by `this.blobManager.summarize(telemetryContext)` on line 1678. This is most likely where your build failures are.